docusaurus.config.js

```diff
@@ -7,9 +7,9 @@ module.exports = {
   title: "Camunda 8 Docs",
   tagline: "Documentation for all components of Camunda 8",
   // url: "https://camunda-cloud.github.io",
-  url: "https://docs.camunda.io",
+  url: process.env.DOCS_SITE_URL || "https://docs.camunda.io",
```

Comment:
one thing to keep in mind: ensure that these changes are compatible with scripts like https://github.com/camunda/camunda-docs/blob/7ca2db8427d1c07d64b9eebf9aeafaaa67c5ee7d/hacks/isolateVersion/7-updateDocusaurusConfig.sh#L6 (maybe there's more places like that, @pepopowitz should know)

Reply:
I will dig how this script is used and try to see whether I can test these changes. If @pepopowitz has any other hints  on the matter they are very much appreciated

Reply:
I string searched and I am under the impression that [/7-updateDocusaurusConfig.sh](https://github.com/camunda/camunda-docs/blob/7ca2db8427d1c07d64b9eebf9aeafaaa67c5ee7d/hacks/isolateVersion/7-updateDocusaurusConfig.sh#L6) is being invoked from the file `allSteps.sh` only. I dont see any instructions on how to run that script and what its purpose is exactly so I am not sure how to proceed here. I will wait for feedback from @pepopowitz

Reply:
Thanks for calling it out -- I will take care of updating this and similar required updates! 

I run this script when we archive/extract an older version of the docs into its own website. It's not really worth updating proactively, as it is run so infrequently, and truly testing it will be best handled when I do the next version archival.

Reply:
@pepopowitz do you have something in the backlog for this? I don't want it to sneak up on us in October.